### PR TITLE
refactor: run provision-react-component

### PR DESCRIPTION
**Pull request by keithamus**
It is recommended that you pull these changes down and test them to ensure that `npm start` and `npm run pages` work as you'd expect (make sure to `npm i`!).

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -31,11 +31,11 @@
   box-shadow: 0 10px 30px rgba(0, 0, 0, 0.19), 0 6px 10px rgba(0, 0, 0, 0.23);
 }
 
-.balloon--not-visible .balloon__link:after {
+.balloon--not-visible .balloon__link::after {
```
Comment (umbobabo) [inline src/index.css]: I think there is an error here

File: src/index.css [hunk 2]
```diff
@@ -31,11 +31,11 @@
...
   display: none;
 }
 
-.balloon--visible .balloon__link:after {
+.balloon--visible .balloon__link::after {
```
Comment (umbobabo) [inline src/index.css]: Here too.

File: src/index.css (hunk 1 above)
Comment (keithamus) [inline src/index.css]: There is not an error here. Pseudo selectors require 2 colons. [Citation](https://css-tricks.com/almanac/selectors/a/after-and-before/)

**Comment by umbobabo** [comment]
Tested on my local, work well. LGTM.

@gcedo  or @fabiosantoscode  please review.

**Comment by gcedo** [comment]
At the moment I have some weird issues on my machine, I don't want to stall the PR. @fabiosantoscode can you review and merge when you're all rainbowish?

**Comment by fabiosantoscode** [comment]
:rainbow: Super-cool

**Pull request merged**